ibm_catalog.json

```diff
@@ -121,17 +132,19 @@
               ]
             },
             {
-              "key": "use_existing_resource_group"
-            },
-            {
-              "key": "resource_group_name"
-            },
-            {
-              "key": "prefix",
-              "required": true
+              "key": "existing_resource_group_name",
+              "required": true,
```

Comment:
should not be force required. The terraform default value is "Default" so no need to add to required inputs

Reply:
same for security enforced variation